script.embuary.helper/default.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python
+
+########################
+
+import xbmcgui
+
+from resources.lib.helper import *
+from resources.lib.utils import *
+
+########################
+
+class Main:
+
+    def __init__(self):
+        self._parse_argv()
+        if self.action:
+            self.getactions()
+        else:
+            xbmcgui.Dialog().ok('Error', 'This is a tool to provide features to a skin and requires skin integration.')
```

Comment:
it would be nice to turn this into localized strings

Reply:
Will be done